code/ui/components/switch/Switch.cs

```diff
@@ -1,14 +1,32 @@
-using Sandbox.UI;
-
 namespace TTTReborn.UI
 {
     public partial class Switch : Sandbox.UI.Switch
     {
-        public Switch(Panel parent = null) : base()
+        public Switch() : base()
         {
-            Parent = parent ?? Parent;
-
             StyleSheet.Load("/ui/components/switch/Switch.scss");
         }
     }
 }
+
+namespace Sandbox.UI.Construct
+{
+    using TTTReborn.UI;
+
+    public static class SwitchConstructor
+    {
+        public static Switch Switch(this PanelCreator self, string className = null, bool enabled = false)
```

Comment:
I probably can't stop myself from bringing this up, but this `className` can be an empty string, since there can't ever be an empty classname.

Reply:
Did you've tested this?

Reply:
Doesn't the default s&box interface take in an empty string when creating a label?

Reply:
nvm leave as is.